metadata.json

```diff
@@ -58,7 +58,7 @@
   "requirements": [
     {
       "name": "puppet",
-      "version_requirement": ">= 4.10.0 < 7.0.0"
+      "version_requirement": ">= 6.11.0 < 7.0.0"
```

Comment:
One of the main use cases of catalog-diff is to help people upgrade from old/unsupported versions of puppet.  It may be counterproductive to break it for Puppet 4 and 5.

Reply:
I updated the README.md to clarify this. Only the system that runs catalog-diff needs to be Puppet 6.11 or newer. You can still get catalogs from ancient Puppetserver/Puppetdb. I think this is fine. Nobody should run this on a box with Puppet Agent 5 or older.